docs/src/getting_started.md

```diff
@@ -48,29 +48,43 @@ python3 -m stata_kernel.install
 
 ### Jupyter
 
-If you chose to install Anaconda you already have Jupyter and Jupyter Lab installed.
+If you chose to install Anaconda you already have [Jupyter Notebook](https://jupyter-notebook.readthedocs.io/en/stable/notebook.html) and [Jupyter Lab](https://jupyterlab.readthedocs.io/en/stable/getting_started/overview.html) installed.
 
-If you downloaded Miniconda, you need to install Jupyter and optionally Jupyter Lab:
+Otherwise, you need to install Jupyter Notebook or Jupyter Lab. I recommend the latter as it is a similar but more modern environment. If you have Miniconda, open the Anaconda Prompt and run:
 
 ```bash
-conda install jupyter
 conda install jupyterlab
 ```
 
-Otherwise you can install them using `pip`:
+If you use pip, you can install it via:
 
 ```bash
-pip install jupyter
-pip install jupyterlab
+pip install jupyterlab 
+pip3 install jupyterlab  # if Python 2 is the default version
 ```
 
+If you would not like to install Jupyter Lab and only need the Notebook, you can install it by running
+
```

Comment:
Actually, this section might not be necessary at all. As this section of the docs is aimed at people new to Jupyter, it might make sense to just provide instructions for installing the lab, which will also install the notebook in turn as it is a dependency of the lab. The text at the beginning still makes it clear that the Notebook is enough in itself.

Having installation instructions or the notebook as well makes the matter a bit clearer, but maybe longer than it needs to be. What do you think?